app/models/target.rb

```diff
@@ -23,6 +23,10 @@
 #  fk_rails_...  (user_id => users.id)
 #
 class Target < ApplicationRecord
+  include ActiveModel::Validations
+
+  validates_with Validators::TargetLimitPerUserValidator
```

Comment:
take a look at our tech guides about the [order of different methods](https://github.com/rootstrap/tech-guides/blob/master/ruby/rails.md#macro-style-methods) inside classes